StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Views/SheetNavigationBar.swift

```diff
@@ -22,10 +22,16 @@ class SheetNavigationBar: UIView {
     static let height: CGFloat = 48
     weak var delegate: SheetNavigationBarDelegate?
     fileprivate lazy var leftItemsStackView: UIStackView = {
-        let stack = UIStackView(arrangedSubviews: [closeButtonLeft, backButton, testModeView])
+        let stack = UIStackView(arrangedSubviews: [dummyView, closeButtonLeft, backButton, testModeView])
         stack.spacing = PaymentSheetUI.defaultPadding
+        stack.setCustomSpacing(PaymentSheetUI.navBarPadding, after: dummyView)
+        stack.alignment = .center
         return stack
     }()
+    fileprivate lazy var dummyView: UIView = {
+        let dummyView = UIView(frame: .zero)
+        return dummyView
+    }()
```

Comment:
Looks good, but if someone see's this in the future they might not understand what this dummyView does. Can we add a comment?